app/components/wallet/add/option-dialog/WalletCreateOptionDialog.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import type { Node } from 'react';
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { defineMessages, intlShape } from 'react-intl';
+
+import Dialog from '../../../widgets/Dialog';
+import DialogCloseButton from '../../../widgets/DialogCloseButton';
+import OptionBlock from '../../../widgets/options/OptionBlock';
+import type { $npm$ReactIntl$IntlFormat } from 'react-intl';
+import styles from '../../../widgets/options/OptionListWrapperStyle.scss';
+
+const messages = defineMessages({
+  dialogTitle: {
+    id: 'wallet.add.optionDialog.create.dialogTitle',
+    defaultMessage: '!!!Create',
+  },
+  createNormalTitle: {
+    id: 'wallet.add.optionDialog.create.normalWallet.title',
+    defaultMessage: '!!!Create a Wallet',
+  },
+  createNormalDescription: {
+    id: 'wallet.add.optionDialog.create.normalWallet.description',
+    defaultMessage: '!!!Choose this option to create your wallet.',
+  },
+  createPaperWalletTitle: {
+    id: 'wallet.add.optionDialog.create.paperWallet.title',
+    defaultMessage: '!!!Paper Wallet',
+  },
+  createPaperWalletDescription: {
+    id: 'wallet.add.optionDialog.create.paperWallet.description',
+    defaultMessage: '!!!If you want to generated a Yoroi paper wallet, you can choose this option to create the funds from your Yoroi paper wallet.',
+  },
+});
+
+type Props = {|
+  +onCancel: void => void,
+  +onPaper: void => void,
+  +onCreate: void => void,
+|};
+
+@observer
+export default class WalletCreateOptionDialog extends Component<Props> {
+  static contextTypes: {|intl: $npm$ReactIntl$IntlFormat|} = {
+    intl: intlShape.isRequired,
+  };
+
+  render(): Node {
+    const { intl } = this.context;
+    const { onCancel, onCreate, onPaper, } = this.props;
+
+    return (
+      <Dialog
+        title={intl.formatMessage(messages.dialogTitle)}
+        closeOnOverlayClick={false}
+        onClose={onCancel}
+        closeButton={<DialogCloseButton />}
+      >
+        <div className={styles.component}>
+          <ul className={styles.optionBlockList}>
+            <OptionBlock
+              parentName="WalletCreateOptionDialog"
+              type="restoreNormalWallet" //TO CHANGE
```

Comment:
TODO? (here and for other option also)
Although I think this is correct

Reply:
Yep, I mean I will change these icons bc they were used for ```restore wallet```, especially the create wallet since wallet paper icon looks great.